### DISCUSSION TITLE
Clipper extension

### DISCUSSION THREAD

**Discussion by sergiorgiraldo** [outline/outline#5735] [General]
I wrote a browser extension for Outline: https://github.com/sergiorgiraldo/OutlineClipper

UI and feature-set are simple, just gets the page, pass thru Readability and saves to a collection. 

Important to change the permissions in the `manifest.json` to enable CORS to your outline API endpoint

**Comment by tommoor** [comment]
Nice! You should post some screenshots or a video so folks can see how it works

**Comment by sergiorgiraldo** [comment]
When you are on the page you want to clip, click on the button for the extension and a new tab opens.

You supply your api key for the first time and it is stored locally in the browser. The extension shows your collections, captures the title as the page name and transforms the readability content into markdown.

![clipper](https://github.com/outline/outline/assets/1144364/435fc20c-c9f7-4615-8eea-b0a04a3c48cf)

**Comment by 0xRoM** [comment]
just wanted to say this is awesome and super useful, thank you :)
(and you spelt "Save" wrong in the settings - it currently says "Svae")
